lib/helpers.ex

```diff
@@ -57,49 +57,69 @@ defmodule GogsHelpers do
     remote_url(git_url, org, repo)
   end
 
+  defp get_org_repo_names(url) do
+    [org, repo] =
+      url
+      |> String.split("/")
+      |> Enum.take(-2)
+
+    {org, repo}
+  end
```

Comment:
Maybe a regex might be better here

Reply:
I tried a regex before I went with this simple pipe code. Unless you're fluent in regex this reads _faster_. 💭